bin/update-ips.bash

```diff
@@ -225,8 +222,14 @@ get_swift_url() {
     exit 1
   fi
 
-  os_token=$(grep -oP "x-subject-token:\s+\K\S+" "${headers}")
-  swift_url=$(echo "${response}" | jq -r '.token.catalog[] | select( .type == "object-store" and .name == "swift") | .endpoints[] | select(.interface == "public" and .region == "'"${swift_region}"'") | .url')
+  {
+    # shellcheck disable=SC2162
+    while read -d $'\r\n' header value; do
```

Comment:
My bad missing this. I don't think there is any reason for not using the `-r` flag.

```suggestion
    while read -r -d $'\r\n' header value; do
```